contracts/container/contract.go

```diff
@@ -296,16 +296,16 @@ func getFromMap(m map[string]any, key string) any {
 	return m[key]
 }
 
-// PutMeta is the same as [Put] (and exposed as put from the contract via
+// PutMeta is the same as [PutNamed] (and exposed as put from the contract via
```

Comment:
does this make impossible to `put` with `metaOnChain`?

Reply:
we should either duplicate code, or be ready to do only one thing in PUTNAMED/PUT pair. _but_ PUT still can be done with PUTNAMED with "", "" name and zone

Reply:
didnt get the answer. With proposed changes, can i `put(container, signature, publicKey, token, true)`?

Reply:
> does this make impossible to put with metaOnChain?

yes, it makes. we either have dozen of methods to overload every case or use the fact that "", "" args make PutNamed behave as Put

> With proposed changes, can i put(container, signature, publicKey, token, true)?

no. but before you were impossible to create a named container with meta-on-chain. now you can `put(container, signature, publicKey, token, "", "", true)` and it is the same as your request but you need to understand it and put such empty args. i was writing code near this method calls when i realized it. now i want that code to be always "put" and nothing more. it simplifies things

Reply:
my proposal is already implemented https://github.com/nspcc-dev/neofs-contract/pull/451#issuecomment-2538854815